csun-metalab/nomi
# NAM-489 -- UI: Fix styling of upload Modal

# Description
Added CSS styling for upload modal.
  
# Testing
1. Visit student page and click the edit button.
2. Ensure the modal meets the requirements and doesn't break at different resolutions.
